local_units/test_views.py

```diff
@@ -133,6 +134,26 @@ def test_detail(self):
         self.assertEqual(response.data["type_details"]["name"], "Code 0")
         self.assertEqual(response.data["type_details"]["code"], 0)
 
+    def test_updated_at_updated_by(self):
```

Comment:
We should also check if updated_at and updated_by are properly updated on mutation.

For eg: We can add a check at the end of this function https://github.com/IFRCGo/go-api/blob/358b89c3efc4d979af41d4ccdda5f315caa67578/local_units/test_views.py#L309 where the updated_by should match the current user and updated_at is not None.

Similarly, we can also do a quick update using another user, then the updated_by should match the new user and updated_at should be greater than the existing updated_at.

> NOTE: I wouldn't worry much about updated_at as it is set by django automatically. So is not None check if fine